.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,212 @@
+name: Integration Tests
+on:
+  schedule:
+    # Run nightly at 2 AM UTC
+    - cron: '0 2 * * *'
```

Comment:
Hmm instead of nightly, we can just run (with approval from a maintainer) on PRs